src/views/text.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Linking, StyleSheet, View } from 'react-native';
 import { Text, Button } from 'react-native-elements';
-import theme from 'react-native-elements/src/config/theme';
+import theme from 'react-native-elements/dist/config/theme';
```

Comment:
Why we need to change this? It was working till now, just asking..

Reply:
did not work for me, pls try cloning the project from scratch

Reply:
I did, working fine for me, I am not sure what you did

Reply:
Working Fine for me without this change

Reply:
I am not sure, i had to change to get it working, It was continuously giving until i made those changes

Reply:
maybe it's just me because it's my first time using the react-native

Reply:
Great! Keep Learing

Reply:
Great! Keep Learning

Reply:
Yeah, You are right,